src/SignaturePad.Forms/SignaturePadView.cs

```diff
@@ -0,0 +1,145 @@
+            set { SetValue(CaptionTextProperty, value); }
+        }
+
+        public Color CaptionTextColor
+        {
+            get { return (Color)GetValue(CaptionTextColorProperty); }
+            set { SetValue(CaptionTextColorProperty, value); }
+        }
+
+        public string ClearText
+        {
+            get { return (string)GetValue(ClearTextProperty); }
+            set { SetValue(ClearTextProperty, value); }
+        }
+
+        public Color ClearTextColor
+        {
+            get { return (Color)GetValue(ClearTextColorProperty); }
+            set { SetValue(ClearTextColorProperty, value); }
+        }
+
+        public string PromptText
+        {
+            get { return (string)GetValue(PromptTextProperty); }
+            set { SetValue(PromptTextProperty, value); }
+        }
+
+        public Color PromptTextColor
+        {
+            get { return (Color)GetValue(PromptTextColorProperty); }
+            set { SetValue(PromptTextColorProperty, value); }
+        }
+
+        public Color SignatureLineColor
+        {
+            get { return (Color)GetValue(SignatureLineColorProperty); }
+            set { SetValue(SignatureLineColorProperty, value); }
+        }
+
+        public float StrokeWidth
+        {
+            get { return (float)GetValue(StrokeWidthProperty); }
+            set { SetValue(StrokeWidthProperty, value); }
+        }
+
+        public Color StrokeColor
+        {
+            get { return (Color)GetValue(StrokeColorProperty); }
+            set { SetValue(StrokeColorProperty, value); }
+        }
+
+        public IEnumerable<Point> Points
+        {
+            get { return GetSignaturePoints(); }
+            set { SetSignaturePoints(value); }
+        }
+
+        /// <summary>
+        /// Returns an image data stream for the current signature.
+        /// (The caller must dispose the stream)
+        /// </summary>
+        /// <param name="imageFormat">The format/encoding of the image that is desired.</param>
+        /// <returns>Returns the image data stream.</returns>
+        public Task<Stream> GetImageStreamAsync(SignatureImageFormat imageFormat)
+        {
+            var args = new ImageStreamRequestedEventArgs(imageFormat);
+            ImageStreamRequested?.Invoke(this, args);
+            return args.ImageStreamTask;
+        }
+
+        private IEnumerable<Point> GetSignaturePoints()
+        {
+            var args = new PointsEventArgs();
+            PointsRequested?.Invoke(this, args);
+            return args.Points;
+        }
+
+        private void SetSignaturePoints(IEnumerable<Point> points)
+        {
+            PointsSpecified?.Invoke(this, new PointsEventArgs { Points = points });
+        }
+
+        private bool RequestIsBlank()
+        {
+            var args = new IsBlankRequestedEventArgs();
+            IsBlankRequested?.Invoke(this, args);
+            return args.IsBlank;
+        }
+
```

Comment:
Instead of events, consider an internal interface.  Not too critical as you made them internal which is good.